### ISSUE TITLE
Fix gt super chest name

### ISSUE THREAD

**Issue by Anarack** [GTNewHorizons/GT-New-Horizons-Modpack#5227]
2.0.8.0 pre
The super chest has disabled in its name, as this is not true anymore this needs to be fixed.

**Comment by Dream-Master** [comment]
gt5u super chest or gt++ super chest?

**Comment by draknyte1** [comment]
![image](https://user-images.githubusercontent.com/3060479/72681366-a1bbf800-3aba-11ea-9f37-0ec7b1debdfa.png)

**Comment by repo-alt** [comment]
So, looks like hiding failed

**Comment by draknyte1** [comment]
Then it's a bug in NEI. ;)
```
	public static void hideItemFromNEI(ItemStack aItemToHide) {
		codechicken.nei.api.API.hideItem(aItemToHide);		
	}
```

**Comment by patrick-hill** [comment]
Why the need for the suffix to begin with if the item is fully enabled?

This is a confusing issue, I was worried the item was bugged so didn't craft it. Thankfully got more insight on the community Discord but still, would rather see the suffix removed as this item isn't actually disabled, right?

**Comment by draknyte1** [comment]
The GT++ super chests *ARE* and shouldn’t be visible in NEI at all. The other ones provided have no suffix and are enabled.

**Comment by patrick-hill** [comment]
Was this fixed after 2.0.8.3 then as in that version, the GT versions continue to show [Disabled] suffix int their names.

**Comment by draknyte1** [comment]
Nope.

**Comment by patrick-hill** [comment]
Okay so either your previous comment is incorrect or I'm confused...

I don't see the GT++ ones in 2.0.8.3, only the GT ones and they DO have the suffix.

**Comment by draknyte1** [comment]
All content added by GT addons is still registered under GT. there’s 32k meta on the ‘machine’ block and we just allocate behaviour to those IDs.

**Comment by chochem** [comment]
> 
> 
> The GT++ super chests _ARE_ and shouldn’t be visible in NEI at all. The other ones provided have no suffix and are enabled.

@draknyte1 No. This is really not what's actually happening in game. There is exactly one type of super chest available. It does work. And it even has a quest. But it does have [disabled] in its name. 

Here is the full description, maybe that lets you identify which one it is:
![2020-03-19_15 50 44](https://user-images.githubusercontent.com/40274384/77086870-42b32a80-69fa-11ea-809d-e857e7ea71bf.png)

**Comment by draknyte1** [comment]
Read the entire issue thanks. @chochem 

As my code shows, mine shouldn’t be visible nor craftable. They would’ve been removed entirely if not for the fact people have them in world and I don’t want them losing their contents.  🤷🏻‍♀️

A Meta ID of 135 means they’re from GTNH’s GT fork, as my IDs are 750-999 & then 2k between 29-31k or 30-32k. (Don’t recall which)

**Comment by chochem** [comment]
thanks, I read everything. At least your answer was partly constructive. So @patrick-hill was correct and this is indeed the GT super chest. So why are you claiming they don't have a suffix when they clearly do?

**Comment by draknyte1** [comment]
Why would anyone have ever added said suffix to the ones rolled into GT? 🤔

**Comment by repo-alt** [comment]
Not very hard to verify that there is no [Disabled] string in GT sources at all.

**Comment by draknyte1** [comment]
Figured out the cause. Fixed in 9.0.
